src/main/target/FF_FORTINIF4/config.c

```diff
@@ -0,0 +1,30 @@
+/*
+ * This file is part of Cleanflight.
+ *
+ * Cleanflight is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Cleanflight is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with Cleanflight.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include <stdbool.h>
+#include <stdint.h>
+
+#include <platform.h>
+
+#ifdef TARGET_CONFIG
+#include "telemetry/telemetry.h"
+
+void targetConfiguration(void)
+{
+    telemetryConfigMutable()->telemetry_inverted = false;
```

Comment:
If you look at https://github.com/betaflight/betaflight/blob/master/src/main/telemetry/telemetry.c#L60 you will see that the code that was added here does not change anything, since `telemetry_inverted` is already initialised to `false`. If they want their boards to work with uninverted telemetry out of the box (which is a customer unfriendly move in my opinion, since it is only designed to make it harder for customers to mix and match equipment from different manufacturers), then `telemetry_inverted` should be `true` (i.e. the standard for SBus is inverted serial, so 'inverted SBus' is uninverted serial).